app/assets/images/payment_icons/latitude_creditline_au.svg

```diff
@@ -0,0 +1 @@
or="#333"/><stop offset=".631" stop-color="#2e2e2e"/><stop offset=".789" stop-color="#1f1f1f"/><stop offset=".961" stop-color="#070707"/><stop offset="1"/></linearGradient><g clip-path="url(#c)"><g clip-path="url(#d)"><path d="M-44.618.956h38v23.975h-38z" fill="url(#a)"/></g><g clip-path="url(#e)"><path d="M-44.618-8.044h38v23.975h-38z" fill="url(#b)"/></g></g><g clip-path="url(#f)"><path d="M0 .022h38v23.957H0z" fill="url(#g)"/><path d="M16.48 10.1v1.1a4.23 4.23 0 00-.774-.069c-1.178 0-1.974.617-1.974 1.645 0 .953.661 1.559 1.974 1.559a3.827 3.827 0 001.377-.211v-4.35h-.294a.325.325 0 00-.309.326zm0 3.81a1.951 1.951 0 01-.738.12c-.987 0-1.392-.514-1.392-1.3a1.3 1.3 0 011.483-1.33 3.682 3.682 0 01.646.052zm1.41-2.73h-.29v3.079h.61v-2.754a.327.327 0 00-.321-.327zm-.162-.349l.43-.189c.07-.031.07-.081 0-.112l-.43-.188a.084.084 0 00-.127.083v.323a.084.084 0 00.126.081zm-8.167.269a5.123 5.123 0 00-1.661.212v2.941h.3a.326.326 0 00.308-.315v-2.417a2.478 2.478 0 011.207-.057l.156-.354c-.061-.01-.203-.01-.31-.01zm2.028 0a1.593 1.593 0 00-1.811 1.605c0 1.108.831 1.622 2 1.622a3.836 3.836 0 001.349-.211l-.177-.315a3.127 3.127 0 01-1.129.189 1.584 1.584 0 01-1.123-.46 1.41 1.41 0 01-.357-.923h2.95c.002-1.023-.645-1.507-1.702-1.507zm-1.241 1.3a1.129 1.129 0 011.219-1.01c.668 0 1.129.3 1.129 1.012zM6.47 10.6a2.035 2.035 0 01.838.166l.178-.3a2.677 2.677 0 00-1.137-.211 2.05 2.05 0 10.014 4.078 2.729 2.729 0 001.123-.211l-.178-.3a2.086 2.086 0 01-.824.154A1.688 1.688 0 116.47 10.6zm26.583.614h.166v.6h.081v-.6h.166v-.061h-.411zM26.9 11.1a5.211 5.211 0 00-1.527.212v2.941h.611v-2.738a2.855 2.855 0 01.844-.113c.583 0 1.151.159 1.151.622v1.908a.326.326 0 00.308.325h.295v-2.245c-.001-.674-.754-.912-1.682-.912zm4.027 0a1.594 1.594 0 00-1.811 1.605c0 1.108.831 1.622 2 1.622a3.841 3.841 0 001.349-.211l-.122-.217a.167.167 0 00-.185-.051 3.338 3.338 0 01-1 .142 1.585 1.585 0 01-1.123-.46 1.4 1.4 0 01-.356-.923h2.951c0-1.023-.646-1.507-1.704-1.507zm-1.241 1.3a1.13 1.13 0 011.22-1.01c.667 0 1.128.3 1.128 1.012zm1.5-3.084a.368.368 0 00.228-.206v.69a.376.376 0 00.568.372l1.93-.846c.313-.137.313-.362 0-.5l-1.93-.846a.376.376 0 00-.568.372v.68a.368.368 0 00-.228-.206l-1.93-.846a.377.377 0 00-.569.372V9.8a.376.376 0 00.569.372zm2.638 2.366l-.209-.538H33.5v.659h.079v-.5l.2.5h.073l.194-.5v.5h.079v-.659h-.1zm-9.365-.509h-.289v3.079h.61v-2.747a.328.328 0 00-.323-.327zm-2.611-.523a.327.327 0 00-.326-.327h-.312v3.6a.327.327 0 00.309.325h2.238v-.326h-1.911zm-1.993-.328h-.6v.85h-.617v.275h.617v2.09c0 .371.285.776 1.094.776a1.73 1.73 0 00.376-.045v-.291a1.291 1.291 0 01-.206.017c-.433 0-.66-.223-.66-.469v-2.072h.866v-.275h-.866zm4.438.5l.431-.189c.069-.031.069-.081 0-.112l-.431-.188a.085.085 0 00-.128.083v.323a.084.084 0 00.126.09z" fill="#fff"/></g><path d="M0 3v18a3.009 3.009 0 003 3h32a2.946 2.946 0 003-3V3a3.009 3.009 0 00-3-3H3a2.946 2.946 0 00-3 3zm35-2a2.006 2.006 0 012 2v18a2.006 2.006 0 01-2 2H3a2.006 2.006 0 01-2-2V3a2.006 2.006 0 012-2z" fill-rule="evenodd" opacity=".5"/></svg>
```

Comment:
I don't see the result of this icon. I'm seeing a white background with grey border.